### DISCUSSION TITLE
Dropdown content is binding to to popover trigger

### DISCUSSION THREAD

**Discussion by volstas** [huntabyte/shadcn-svelte#1500] [Help] [answered]
So I have a weird issue, where one page has both, a dropdown and a popover. But when I click the dropdown button, the size of it and position is binding to the popover trigger.

In dev mode, if I modify something and get it hot reloaded, it fixes itself. But fresh load always has this issue.

I tried turning off SSR but the issue persisted. Kinda lost on what is happening.

Anything obvious to check or fix before I try to create reproducible thing for a bug report?

**Comment by huntabyte** [comment]
I'd need to see the code to determine what might be happening

**Comment by volstas** [comment, marked as answer]
So I fixed it, the issue was some messed up bits-ui references in a mono repo between projects. Very weird side effect because everything else is working besides this binding. But once I fixed to correctly use the new bits-ui version then it started working.

The issue was specifically in the index.js files of the shadcn components. Had a bad bits-ui import there.
